pkg/server/postgres/postgres.go

```diff
@@ -176,3 +187,92 @@ func (p *PostgresPlugin) DeleteDatabase(db *atlas.Database) error {
 	// delete database
 	return nil
 }
+
+func checkUsers(db *atlas.Database, sqldb *sql.DB) error {
+	//// check if users exist; verify their passwords and roles
+	dbUserMap := make(map[string]string)
+	var u string
+	//getting all users from the database
+	rows, err := sqldb.Query(`SELECT u.usename FROM pg_catalog.pg_user u`)
+	if err != nil {
+		return err
+	}
+	for rows.Next() {
+		if err := rows.Scan(&u); err != nil {
+			return err
+		}
+		dbUserMap[u] = "delete me"
+	}
+	glog.V(4).Info("Usersexist in database are", dbUserMap)
+	users := db.Spec.Users
+	glog.V(4).Info("Users from yaml are", users)
```

Comment:
Users requested from yaml are[{me foo <nil> admin} {you bar <nil> read}]

I am getting above output. Fix it

Reply:
done